crates/subspace-core-primitives/src/objects.rs

```diff
@@ -96,26 +96,61 @@ pub enum PieceObject {
         /// Object hash
         #[cfg_attr(feature = "serde", serde(with = "hex"))]
         hash: Blake3Hash,
-        // TODO: This is a raw record offset, not a regular one
-        /// Offset of the object in that piece
+        /// Regular record offset of the object in that piece (after KZG encoding).
```

Comment:
I'm not sure KZG encoding is a helpful comment. In fact there is no encoding done here at all since these are source pieces, they just have one 0 byte inserted after every 31 useful bytes.